### PR TITLE
Prodip/account management

### PR DESCRIPTION
**Pull request by Prod-IP**

### CODE REVIEW

File: src/controllers/user/update-user-details.controller.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ * @body {name: string, description: string, phone: string, status: string}
+ * @returns {updated count}
+ */
+
+const updateUserDetails = asyncHandler(async (req, res) => {
+  try {
+    const { name, description, phone, status } = req.body as User;
+    const { id } = req.body as { id: string };
+
+    const body = {} as User;
```
Comment (Bijoy-007) [inline src/controllers/user/update-user-details.controller.ts]: @Prodip-Kumar-Paul  Do not typecast an empty object add keys with "" values instead

File: prisma/schema.prisma [hunk 2]
```diff
@@ -31,6 +31,7 @@ model User {
   description String
   teacherId   String
   phone       String
+  status      String    @default("ACTIVE")
```
Comment (Bijoy-007) [inline prisma/schema.prisma:34]: @Prodip-Kumar-Paul Are we going to have more than 2 statuses? If not then why not use a boolean?

**Review by Bijoy-007 — COMMENTED** [review]
@Prodip-Kumar-Paul

File: src/controllers/user/update-user-details.controller.ts (hunk 1 above)
Comment (Prod-IP) [reply src/controllers/user/update-user-details.controller.ts]: check below codes, my logic based on that scenario

how about using this ->  const body: Partial<User> = {};

Comment (Prod-IP) [reply src/controllers/user/update-user-details.controller.ts]: @Bijoy-007

File: prisma/schema.prisma (hunk 2 above)
Comment (Prod-IP) [reply prisma/schema.prisma:34]: yup better approach

**Pull request merged**